rapidast.py

```diff
@@ -154,15 +154,27 @@ def run():
         f"log level set to debug. Config file: '{parser.parse_args().config_file}'"
     )
 
+    # Load config file
     try:
         config = configmodel.RapidastConfigModel(
             yaml.safe_load(load_config_file(parser.parse_args().config_file))
         )
     except yaml.YAMLError as exc:
         raise RuntimeError(
-            f"Something went wrong while parsing one of the config '{parser.parse_args().config_file}':\n {str(exc)}"
+            f"YAML error in config {parser.parse_args().config_file}':\n {str(exc)}"
         ) from exc
 
+    # Optionally adds default if file exists (will not overwrite existing entries)
```

Comment:
RE: `(will not overwrite existing entries)` => does it mean, if we run a scan with a config file that contains `general.container.type = "podman"`,  then having the container_default_config.yaml(of setting it to 'none') won't affect?   IMO, for a specific case of running a scan with the RapiDAST image, what we want to do is overwrite it to 'none' regardless of whether the original config being set  to 'podman' or 'none'?

Reply:
Correct: the config file is as considered higher priority: if a config file has `type: podman`, podman will always be run no matter what. For this to work, the config file needs to rely on default.
I am not sure I like the idea of letting the program transgress on a user choice.

However, maybe we can create a convertion command line, using a tool like `yq`, to parse and change the config files from "podman" to "none", and run that command line in the Containerfile ? this way, while copying any Containerfile as it is doing at the moment, it would also modify them to run with `none` ? (or remove the `type` entirely, and rely on the default one